### PR TITLE
h5p listing page improvements

### PR DESCRIPTION
**Pull request by arzola**
This PR solves https://github.com/pressbooks/pressbooks/issues/1996

We have a h5p-listing page on pressbooks, it's a page where we can see the total of h5p activities of a book the current implementation was printing all the activities in the first page load this is very exhaustive for server performance I added pagination to get chunks of 20 activities per page.

### How to test

1. Add a couple of h5p activities in your book (or clone https://integrations.pressbooks.network/h5p-clone, which has 30+ H5P activities in it)
2. Visit `https://pressbooks.test/yourbook/h5p-listing` the h5p listing page
3. You should see the total of activities under the page title
4. If you want to test the pagination you can change the default `per_page` parameter of the `inc/helpers/namespace.php` `get_h5p_activities` function

Notes:

I removed the transient caching because I think is no more needed with the pagination because the queries are going to be fast

### CODE REVIEW

**Comment by SteelWagstaff** [comment]
@os and @richard015ar I made a friendly commit with some changes to DOM structure and styling so that page looks like this: 
screencapture-pressbooks-test-h5ptest-h5p-listing-2021-07-28-15_17_37. Love the new improvements, Oscar!

File: page-h5p-listing.php [hunk 1]
```diff
@@ -1,41 +1,46 @@
...
 
-				foreach ( $activities as $activity ) {
+				foreach ( $data['activities'] as $activity ) {
 					$short_code = sprintf( '[h5p id="%s"]', $activity['ID'] );
 					echo '<tr>';
 					echo sprintf( '<td>%s</td>', $activity['ID'] );
 					echo sprintf( '<td>%s</td>', $activity['title'] );
 					echo sprintf( '<td>%s</td>', $activity['activity_type'] );
-					echo sprintf( '<td><button h5p-id="%s" type="button" show-activity-text="%s" hide-activity-text="%s" class="btn btn-secondary btn-sm h5p-row-item"></button></td>', $activity['ID'], __( 'View activity', 'pressbooks-book' ), __( 'Hide activity', 'pressbooks-book' ) );
+					echo sprintf( '<td><button h5p-id="%s" type="button" show-activity-text="%s" hide-activity-text="%s" class="btn btn-secondary btn-sm h5p-row-item"></button></td>', $activity['ID'], __( 'Show activity', 'pressbooks-book' ), __( 'Hide activity', 'pressbooks-book' ) );
```
[the remark this answers is not in the document]
Comment (SteelWagstaff) [reply page-h5p-listing.php:33]: We should probably add aria-lables for these buttons: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#associated_aria_roles_states_and_properties

[the remark this answers is not in the document]
Comment (richard015ar) [reply page-h5p-listing.php:33]: @arzola changes LGTM, only pending thing is the accesslint suggestion

**Comment by arzola** [comment]
@SteelWagstaff wow, really nice improvements thanks for that

**Review by richard015ar — APPROVED** [review]
Thank you Os, you rocked!

**Pull request merged**